web/src/components/filings-calendar/FilingsCalendarSingleGrid.test.tsx

```diff
@@ -411,4 +425,34 @@ describe("<FilingsCalendarSingleGrid />", () => {
       screen.queryByText(Config.dashboardDefaults.viewLessFilingsButton),
     ).not.toBeInTheDocument();
   });
+
+  it("renders an xray renewal event", () => {
```

Comment:
create a test where the renewal expired yesterday, i dont think it will render. Create it within FilingsCalendar.tsx.